src/main/fc/fc_core.c

```diff
@@ -382,6 +382,10 @@ void tryArm(void)
         ENABLE_ARMING_FLAG(ARMED);
         ENABLE_ARMING_FLAG(WAS_EVER_ARMED);
 
+#ifdef USE_ACRO_TRAINER
+        pidAcroTrainerInit();
```

Comment:
Is this still needed, considering that `pidAcroTrainerInit()` will be called every time the trainer mode is enabled by switch?

Reply:
Yes, this is still required and very important. Imagine the scenario where the pilot has the mode enabled before connecting the battery.  While securing the battery and placing the quad on the line it's possible that they exceed the angle limit and cause the state to latch preventing further rotation in that direction until the stick reverses.  Without resetting the state on arming we could get into a situation where the pilot may not be able to rotate an axis in one direction initially on takeoff.

Reply:
> Imagine the scenario where the pilot has the mode enabled before connecting the battery.

That is a pretty perpetuum mobile-y scenario. :stuck_out_tongue_winking_eye: 

But good point about the latching behaviour.